### ISSUE TITLE
Make email optional

### ISSUE THREAD

**Issue by xaur** [decred/dcrstakepool#274]
Only 20 stakepools know email addresses of people holding some 50% tickets. I don't know exact percentage, but it is big enough to be a concern.

Email is obsolete, legacy, ancient, anti-privacy comm system. It must not be a requirement for new systems built in 21 century to support the new decentralized finance.

Stakepools shall allow what I call "first party registration", where one is not forced into _any_ 3rd party to sign up with a service. Let users have a choice on their account recovery.

Please add 'privacy' label.
Similar to decred/politeia#554.
Related to #259.

**Comment by mverrilli** [comment]
I've thought about this a bit as well. Seems to me that you could make your wallet your key where you simply prove who you are via signature.

**Comment by isuldor** [comment]
The dex proposal could result in code that would be useful for accomplishing this. Namely, acquiring a psuedo-account with the vsp by paying a small fee, rather than using email or even credentials. More work would still need to be done (importing redeem scripts, etc), but that would be a strong way to start.

**Comment by xaur** [comment]
Yes, that would be public key based authentication (#291). As I understand, developers avoid novel authentication schemes to cut costs on research and possible rabbit holes.

But that's not required to drop the dinosaur email requirement. Just allow to use login+password like in good old times.
